File: doc/src/scripts/RemoteTechExample.java
import krpc.client.Connection;
import krpc.client.RPCException;
import krpc.client.services.RemoteTech;
import krpc.client.services.RemoteTech.Antenna;
import krpc.client.services.RemoteTech.Comms;
import krpc.client.services.SpaceCenter;
import krpc.client.services.SpaceCenter.Part;
import krpc.client.services.SpaceCenter.Vessel;

import java.io.IOException;

public class RemoteTechExample {
    public static void main(String[] args) throws IOException, RPCException {
        Connection connection = Connection.newInstance("RemoteTech Example");
        SpaceCenter sc = SpaceCenter.newInstance(connection);
        RemoteTech rt = RemoteTech.newInstance(connection);
        Vessel vessel = sc.getActiveVessel();

        // Set a dish target
        Part part = vessel.getParts().withTitle("Reflectron KR-7").get(0);
        Antenna antenna = rt.antenna(part);
        antenna.setTargetBody(sc.getBodies().get("Jool"));

        // Get info about the vessels communications
        Comms comms = rt.comms(vessel);
        System.out.println("Signal delay = " + comms.getSignalDelay());
    }
}

File: doc/src/scripts/RemoteTechExample.cs
using System;
using KRPC.Client;
using KRPC.Client.Services.RemoteTech;
using KRPC.Client.Services.SpaceCenter;

class RemoteTechExample
{
    public static void Main ()
    {
        var connection = new Connection ("RemoteTech Example");
        var sc = connection.SpaceCenter ();
        var rt = connection.RemoteTech ();
        var vessel = sc.ActiveVessel;

        // Set a dish target
        var part = vessel.Parts.WithTitle ("Reflectron KR-7") [0];
        var antenna = rt.Antenna (part);
        antenna.TargetBody = sc.Bodies ["Jool"];

        // Get info about the vessels communications
        var comms = rt.Comms (vessel);
        Console.WriteLine ("Signal delay = " + comms.SignalDelay);
    }
}
